src/frontend/src/flows/manage/index.ts

```diff
@@ -139,7 +139,7 @@ const pageContent = (userNumber: bigint, devices: DeviceData[]) => html`
     <label>Identity Anchor</label>
     <div class="highlightBox">${userNumber}</div>
     <div class="labelWithAction">
-      <label class="labelAddedDevices" id="deviceLabel"
+      <label class="labelAddedDevices" class="deviceLabel"
```

Comment:
Nit: I don't think there is any styling attached to the `deviceLabel` class. You could also just remove it?